src/main/java/com/gluonhq/richtextarea/action/ActionFactory.java

```diff
@@ -1,11 +1,7 @@
 package com.gluonhq.richtextarea.action;
 
 import com.gluonhq.richtextarea.RichTextArea;
-import com.gluonhq.richtextarea.model.Decoration;
-import com.gluonhq.richtextarea.model.Document;
-import com.gluonhq.richtextarea.model.ImageDecoration;
-import com.gluonhq.richtextarea.model.TableDecoration;
-import com.gluonhq.richtextarea.model.TextDecoration;
+import com.gluonhq.richtextarea.model.*;
```

Comment:
Avoid the use of wildcards (set IntelliJ->Preferences->Editor->Code Style->Java->Class count to use import with *: 99 or alike).